rust/src/progress.rs

```diff
@@ -133,6 +135,35 @@ impl ProgressState {
     }
 }
 
+/// Compute the maximum number of digits needed to represent an integer when
+/// formatted as decimal.
+fn n_digits(n: u64) -> u32 {
```

Comment:
Heh https://github.com/jlebon/git-bstatus/blob/6b877bfea950d4a24ebd73826f5f1ada04148246/src/utils.rs#L61.

Reply:
Guess there aren't that many ways to count the number of digits. :)